test/utils/browser.js

```diff
@@ -81,6 +144,11 @@ exports.assertBrowser = function () {
   });
 };
 
+exports.simpleExecute = function (fn, args, cb) {
+  // DEV: Barebones execute function on the browser, this works when .execute does not
+  this.browser.execute(functionToString(fn).body, args, cb);
+};
```

Comment:
@twolfson I have determined that the logic in `execute` does not fire the `before` function when called from within a `before` function, this kinda makes it hard to use.  Whereas the `simpleExecute` logic will be called no matter what.  Personally I think `simpleExecute` should stay around as it could be helpful in future tests and it fills in a use case that `execute` does not provide?